### DISCUSSION TITLE
Reload property inside proccessor

### DISCUSSION THREAD

**Discussion by samar-elsayed** [apache/camel-karavan#1448] [Q&A] [answered]
### Describe the bug

I am using the below to access the property from application.properties from the context inside the **proccessor**

`exchange.getContext().resolvePropertyPlaceholders("{{property}}")`

how can I reload this property automatically if it has been changed inside application.properties ? 

I have tried `camel.main.context-reload-enabled = true` but the property change wont reflected/reloaded

### Steps to reproduce the behavior

same as above

### Variant

Vs Code Extension

### Container Management (if applicable)

None

### Operating System (if applicable)

None

### Version

4.8.0

**Comment by mgubaidullin** [comment, marked as answer]
application.properties file should not change in runtime
